lib/u3d/iniparser.rb

```diff
@@ -66,6 +67,17 @@ def load_ini(version, cached_versions, os: U3dCore::Helper.operating_system, off
         result
       end
 
+      def create_linux_ini(version, size)
+        ini_name = INI_NAME % { version: version, os: 'linux' }
+        Utils.ensure_dir(default_ini_path)
+        ini_path = File.expand_path(ini_name, default_ini_path)
+        unless File.file?(ini_path)
+          File.open(ini_path, 'wb') do |f|
+            f.write "[Unity]\ntitle=Unity\nsize=#{size}"
```

Comment:
Should we add a comment or a custom INI field to indicate this is our file in case Unity starts supporting them and we start downloading them?